releases/release-1.31/release-notes/maps/pr-123587-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 123587
+releasenote:
+  text: Fix "-kube-test-repo-list" e2e flag may not take effect
```

Comment:
I am not sure if I understand this. Did they fix the flag which was not working before?

Reply:
This is a e2e test bug fix. The global variable initialization was executed before the flag parsing, so the e2e test variable only stored the default image, instead of the image specified in flag. It's not a user facing change though, so I'm surprised they added a release note.